### ISSUE TITLE
amd build appears to have ended prematurely.

### ISSUE THREAD

**Issue by jdgh000** [pytorch/pytorch#113395]
[Uploading python-amd-build.log…]()
### 🐛 Describe the bug

CPU based build worked OK till finish. However when following AMD build isnruction part, it appears to have stopped/failed prematurely. Worst part is no log/error message files very minimal indication of failure: "ninja: build stopped: subcommand failed". 

python tools/amd_build/build_amd.py
pyrhon setup.py develop.

[5602/6366] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Operators_0.cpp.o
[5603/6366] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Operators_1.cpp.o
[5604/6366] Building CXX object third_party/fbgemm/CMakeFiles/fbgemm_avx2.dir/src/FbgemmI8Depthwise3DAvx2.cc.o
[5605/6366] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Operators_2.cpp.o
[5606/6366] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/RegisterCPU.cpp.o
[5607/6366] Building CXX object third_party/fbgemm/CMakeFiles/fbgemm_avx2.dir/src/FbgemmI8DepthwiseAvx2.cc.o
ninja: build stopped: subcommand failed.
Building wheel torch-2.2.0a0+git63fc482
-- Building version 2.2.0a0+git63fc482
cmake -GNinja -DBUILD_PYTHON=True -DBUILD_TEST=True -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/root/gg/git/pytorch-build/pytorch-amd-build/torch -DCMAKE_PREFIX_PATH=/usr/lib/python3.9/site-packages -DNUMPY_INCLUDE_DIR=/usr/local/lib64/python3.9/site-packages/numpy/core/include -DPYTHON_EXECUTABLE=/usr/bin/python -DPYTHON_INCLUDE_DIR=/usr/include/python3.9 -DPYTHON_LIBRARY=/usr/lib64/libpython3.9.so.1.0 -DTORCH_BUILD_VERSION=2.2.0a0+git63fc482 -DUSE_NUMPY=True /root/gg/git/pytorch-build/pytorch-amd-build
cmake --build . --target install --config Release

### Versions

torch main branch

cc @malfet @seemethere @jeffdaily @sunway513 @jithunnair-amd @pruthvistony @ROCmSupport @dllehr-amd @jataylo @hongxiayang

**Comment by malfet** [comment]
Can you share a full build log please? As "ninja: build stopped: subcommand failed" means one of the build commands executed in parallel failed, but it's not necessarily the last one. If you want to run the build sequentially define `MAX_JOBS=1` environment variable before kicking the build.

**Comment by hongxiayang** [comment]
based on my experience, the full log should show some errors. @jdgh000 Did you upload the log as requested by @malfet ? Do you still have the problem ?

**Comment by hongxiayang** [comment]
We are going to close it if no more information is provided.

**Comment by malfet** [comment]
@hongxiayang usually policy for `needs reproduction` label is to wait for two weeks and then close. Looks like to weeks have passed, so it's safe to close.
@jdgh000 please do not hesitate to open a new issue or upload logs to this one and reopen it.
